cocoa/tests_backend/widgets/table.py

```diff
@@ -144,3 +145,9 @@ async def activate_row(self, row):
             delay=0.1,
             clickCount=2,
         )
+
+    async def acquire_keyboard_focus(self):
+        self.native_table.window.makeFirstResponder(self.native_table)
```

Comment:
Yes, because as I'd previously mentioned, toga.Table.focus() is currently a no-op for some reason, it's been explicitly disabled in core/src/toga/widgets/table.py
```
    def focus(self) -> None:
        """No-op; Table cannot accept input focus."""
        pass
```
I was planning to open an issue about this as I, as well, don't understand why table.focus has been made into a no-op, it makes it so I can't focus my users on a list programmatically which I wanted to do since a giant list was the primary control of my app.

Reply:
Ah - my apologies - I forgot you'd raised that. 

I can't give you a good answer for why it's been disabled; my guess is that there might have been some odd behavior related to selection handling. That's definitely worth a standalone issue (and investigation); if you can fix it, that would definitely be welcome. I can't see any fundamental reason why a table shouldn't be able to be given focus, as long as the behaviour otherwise well defined.

Reply:
Also - that issue doesn't need to be fixed to land this PR; but it might be worth dropping a comment on the "pseudo-focus" statements highlighting that they *could* be replaced with `self.widget.focus()` once the issue is resolved (adding a reference to the new ticket number for that issue).